src/components/FoodList/FoodList.js

```diff
@@ -1,28 +1,51 @@
-import foodData from '../../data/foodData.json'
 import styled from 'styled-components/macro'
+import { useState, useEffect } from 'react'
+import getFood from '../../services/getFood'
 
 export default function FoodList() {
+  const [foodList, setFoodList] = useState([])
+  const [searchInput, setSearchInput] = useState('')
+
+  useEffect(() => {
+    getFood()
+      .then((data) => setFoodList(data))
+      .catch((error) => console.log(error))
```

Comment:
You could add an error message instead of the `console.log`, but maybe you intended this task for later.